### PR TITLE
[stable/traefik]: add configurable host ports for deployments

### PR DESCRIPTION
**Pull request by Art3mK**
#### What this PR does / why we need it:

This PR allows user to specify host ports for http/https/dashboard container ports

#### Checklist

- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by pho-enix** [comment]
Similar to:
#11164
#11209

File: stable/traefik/templates/deployment.yaml [hunk 1]
```diff
@@ -118,7 +118,7 @@ spec:
         - name: http
           containerPort: 80
           {{- if .Values.deployment.hostPort.httpEnabled }}
-          hostPort: 80
+          hostPort: {{ .Values.deployment.hostPort.httpPort }}
```
Comment (dtomcej) [inline stable/traefik/templates/deployment.yaml]: These should have a default

File: stable/traefik/values.yaml [hunk 2]
```diff
@@ -354,6 +354,9 @@ deployment:
     httpEnabled: false
     httpsEnabled: false
     dashboardEnabled: false
+    httpPort: 80
```
Comment (dtomcej) [inline stable/traefik/values.yaml]: These should be optional (commented out by default)

**Review by dtomcej — CHANGES_REQUESTED** [review]

File: stable/traefik/templates/deployment.yaml (hunk 1 above)
Comment (Art3mK) [reply stable/traefik/templates/deployment.yaml]: fixed

File: stable/traefik/values.yaml (hunk 2 above)
Comment (Art3mK) [reply stable/traefik/values.yaml]: fixed

**Comment by Art3mK** [comment]
@dtomcej, I've added suggested changes

**Comment by Art3mK** [comment]
ping @dtomcej

**Comment by Art3mK** [comment]
those feedback times are killing me, ping @dtomcej :)

**Pull request merged**